test-it/karma/propertynode/PropertyNodeSequential.Test.js

```diff
@@ -0,0 +1,83 @@
+describe('The Vireo PropertyNode', function () {
+    'use strict';
+
+    var Vireo = window.NationalInstruments.Vireo.Vireo;
+    var vireoRunner = window.testHelpers.vireoRunner;
+    var fixtures = window.testHelpers.fixtures;
+    var spyHelpers = window.testHelpers.spyHelpers;
+
+    var vireo, spy, runSlicesAsync;
+
+    var propertyNodeReadRead = fixtures.convertToAbsoluteFromFixturesDir('propertynode/ReadRead.via');
+    var propertyNodeWriteRead = fixtures.convertToAbsoluteFromFixturesDir('propertynode/WriteRead.via');
+    var propertyNodeWriteWrite = fixtures.convertToAbsoluteFromFixturesDir('propertynode/WriteWrite.via');
+    var propertyName = 'Value';
+
+    beforeAll(function (done) {
+        fixtures.preloadAbsoluteUrls([
+            propertyNodeReadRead,
+            propertyNodeWriteRead,
+            propertyNodeWriteWrite
+        ], done);
+    });
+
+    beforeEach(function () {
+        vireo = new Vireo();
+        spy = jasmine.createSpy();
+    });
+
+    it('sequential read read operations invoke JavaScript callbacks with correct parameters', function (done) {
+        var viName = '%3A%3AWeb%20Server%3A%3AInteractive%3A%3AWebApp%3A%3AReadRead%2Egviweb';
```

Comment:
Could you change the vi name so its just ReadRead or similar?

Reply:
I didn't want to do that since the size of allocated strings mattered in order to reproduce the issue. VI name might no be related but it was difficult to reproduce once you changed the name of a control or in some cases just adding some dummy code at the beginning of the .via file.